test/mpi/include/mtest_dtp.h

```diff
@@ -0,0 +1,318 @@
+/*
+ * Copyright (C) by Argonne National Laboratory
+ *     See COPYRIGHT in top-level directory
+ */
+
+#ifndef MTEST_DTP_H_INCLUDED
+#define MTEST_DTP_H_INCLUDED
+
+#include "mtest_common.h"
+
+enum {
+    MTEST_DTP_ONCE,
+    MTEST_DTP_PT2PT,
+    MTEST_DTP_COLL,
```

Comment:
when would you use this vs the other COLL types?

Reply:
This one is for "dtp" tests, e.g. `bcast.c`, which need parameters e.g. `-seed`, `-testsize`, `-type`, etc.
`MTEST_DTP_COLL_COUNT/NOCOUNT` is for non-dtp collective tests that only need extra `memtype` argument.

Reply:
I see. It guess it is just a bit odd to see an `MTEST_DTP_` prefix for non-dtpools usage.

Reply:
It is a bit odd. How about `MTEST_COLL_COUNT/NOCOUNT`?

Reply:
Sure, that works for me.

Reply:
fixed.